docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3'
+
+services:
+  base:
+    build: .
+    image: "terra-clinical:${BRANCH:-latest}"
+  dev:
+    image: "terra-clinical:dev"
+    build:
+      context: ./
+      args:
+        - npm_install=false
+    ports:
+      - 8080:8080
+    volumes:
+      - .:/opt/module
+  standalone-chrome:
+    image: 'selenium/standalone-chrome:3.11.0-californium'
```

Comment:
Where you still going to create a terra selenium-docker image to dry this up? 

If we do, would need to have an TZ options for downstream passivity in consuming projects

Reply:
As of now, no I'm not drying this up with a terra selenium docker image. I'm not convinced the added maintenance is worth drying up 3 lines of code.